backend/aws.go

```diff
@@ -312,7 +312,7 @@ func pollForASG() (instances []virtualMachine, err error) {
 
 			isValidASG := false
 			for _, tag := range asg.Tags {
-				if *tag.Key == "power-toggle-enabled" && *tag.Value == "true" {
+				if *tag.Key == requiredTagKey && *tag.Value == "true" {
```

Comment:
please also use the var `requiredTagValue` instead of "true"

Reply:
fair point, thank you, i missed that